test/test-feel.js

```diff
@@ -63,7 +66,14 @@ export function fileTests(file, fileName, mayIgnore = defaultIgnore) {
       );
     }
 
-    let [ , name, configStr ] = /(.*?)(\{.*?\})?$/.exec(m[1]);
+
+    const titleMatch = /(.*?)(\{.*?\})?$/.exec(m[1]);
```

Comment:
This made my eyebrows rise, but I guess it's still OK to have such regexes in the test suite.